packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -74,6 +74,7 @@ class DebuggerController extends DisposableController
     }
     if (currentScriptRef.value != null) {
       await programExplorerController.selectScriptNode(currentScriptRef.value);
+      programExplorerController.resetOutline();
```

Comment:
do we want to reset the outline each time selectScriptNode is called? If that's the case, I'm wondering if we should put a call to resetOutline inside of selectScriptNode